dataset.go

```diff
@@ -366,6 +366,13 @@ func (set dataset) CopyNTimes(from Data, fromRow, toRow int, dup []int) {
 	}
 }
 
+func (set dataset) CopyByIndexes(from Data, fromRows []int, toRow int) {
+	src := from.(dataset)
+	for i, idx := range set {
```

Comment:
it was nicer before :)
```suggestion
	for i, d := range set {
```